### ISSUE TITLE
Consider adding metric name to page title

### ISSUE THREAD

**Issue by brizental** [mozilla/glean-dictionary#2124]
It just looks weird when you put a glean dicitonary link in Slack or Jira because they sub it for the page title and the page title is always Glean Dictionary. See: https://mozilla-hub.atlassian.net/browse/VPN-5269?focusedCommentId=831140

**Comment by Iinh** [comment]
Yeah it doesn't look great in JIRA I noticed. Wondering where JIRA sources their page title from, since the dictionary does have dynamically generated page title, e.g. metric/ping name.

![CleanShot 2024-02-09 at 17 09 45@2x](https://github.com/mozilla/glean-dictionary/assets/28797553/4f370958-c8d8-4e14-9690-fe5624bc36b1)

**Comment by brizental** [comment]
Huh, there you go it's a Jira bug :facepalm:

**Comment by wlach** [comment]
The problem is that the title is generated dynamically after the page loads. Moving to SvelteKit would fix this, but that'd be a large task: https://github.com/mozilla/glean-dictionary/issues/453

Alternately, you could generate a bunch of index.html's corresponding to each path with the title (and possibly other metadata) set in each document. Also kind of painful, but probably easier.
